PC/layout/support/pymanager.py

```diff
@@ -82,7 +81,7 @@ def calculate_install_json(ns, *, for_embed=False, for_test=False):
     # Tag shown in 'py list' output
     DISPLAY_TAG = f"{XY_TAG}-dev{TAG_ARCH}" if VER_SUFFIX else XY_ARCH_TAG
 
-    DISPLAY_SUFFIX = ", ".join(i for i in DISPLAY_TAGS if i)
+    DISPLAY_SUFFIX = ", ".join(filter(None, DISPLAY_TAGS))
```

Comment:
This change could go back and forth all day, depending on whoever is updating it to their preference. `filter(None` in particular is a bit of a trick (unless you know it, you have to look it up), which is why I prefer the comprehension style.

Reply:
```suggestion
    DISPLAY_SUFFIX = ", ".join(i for i in DISPLAY_TAGS if i)
```